macOS/DuckDuckGo/Application/AppDelegate.swift

```diff
@@ -147,7 +147,7 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 
 #if SPARKLE
     var updateController: UpdateController!
-    var dockCustomization: DockCustomization!
+    var dockCustomization: DockCustomization?
```

Comment:
Converting this to optional to safeguard that if somebody is accessing the object before it was initialized, they will need to check if it is initialized.